manual/source/release.rst

```diff
@@ -4,6 +4,22 @@ Release notes
 =============
 
 
+.. _release-notes-1.119:
+
+Release 1.119.0
+---------------
+
+Other changes
+.............
+
+#. On FreeBSD and Linux, if the MPS handles a signal while the client
+   program is blocked in a system call, the system call is
+   automatically restarted and does not fail with ``EINTR``. See
+   `GitHub issue #9`_.
```

Comment:
A better link would be to "Signal and exception handling issues" in the reference manual.